functions/src/api.ts

```diff
@@ -122,18 +107,21 @@ export const validate = <T extends z.ZodTypeAny>(schema: T, val: unknown) => {
   }
 }
 
+const DEFAULT_OPTS: HttpsOptions = {
+  minInstances: 1,
+  cors: [CORS_ORIGIN_MANIFOLD, CORS_ORIGIN_LOCALHOST],
```

Comment:
Cool! This looks simpler than before.

Reply:
Yes, including CORS control is new functionality in v2 Cloud Functions, so before we had to do it ourselves, but now we don't.